app/helpers/barnardos/action_view/form_helpers.rb

```diff
@@ -90,6 +91,14 @@ def radio_group_vertical(name, legend, selection_list, value = nil, error: nil,
                 end
               )
             end
+
+            concat(
+              content_tag(:div,
+                          "#{name.to_s.humanize} #{error[0]}",
```

Comment:
Should this just be a string at this stage? We have the singular `error` but we're passing an array.